src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -33,6 +33,10 @@ class YarnBuilder extends AbstractBuilder {
     yarn("sonar-scan ${properties}")
   }
 
+  def highLevelDataSetup(String dataSetupEnvironment) {
+    //TODO: not implemented yet
```

Comment:
this seems to be important?

Teams are normally using node in their repo per definition pattern as far as I know, e.g.:

https://github.com/hmcts/finrem-ccd-definitions

Reply:
solved in [b7ebe7a5e40a9e22fc3300f30a00582f0183cfb1](https://github.com/hmcts/cnp-jenkins-library/pull/747/commits/b7ebe7a5e40a9e22fc3300f30a00582f0183cfb1)